main.py

```diff
@@ -74,24 +74,24 @@ def process_arguments(args):
     """
     # Define the transformations for each argument
     transformations = {
-        "backdate_pools": bool,
+        "backdate_pools": lambda x: x.lower() == "true",
```

Comment:
If you use `lambda x: f'{x}'.lower() == 'true'`, then you can keep all of the assignment below in their natural format.

In other words, you can continue using `someVar = False` instead of replacing it with `someVar = 'False'`, etc.

Regardless of whether or not you accept this suggestion, it might be better to implement something like:
```
is_true = lambda x: x.lower() == "true"
```
Or even:
```
def is_true(x):
    return x.lower() == "true"
```
And then use `is_true` instead of all of these lambdas.

Reply:
changed to `is_true` approach, thanks